pages/linux/krita.md

```diff
@@ -0,0 +1,25 @@
+# krita
+
+> Krita is a sketching and painting program designed for digital artists.
+> See also: `gimp`.
+> More information: <https://docs.krita.org/en/reference_manual/linux_command_line.html>.
+
+- Start krita:
+
+`krita`
+
+- Start without the splash screen:
+
+`krita --nosplash`
+
+- Open the specified files:
+
+`krita {{path/to/image_file1.ext path/to/image_file2.ext ...}}`
```

Comment:
```suggestion
`krita {{path/to/image_file1 path/to/image_file2 ...}}`
```

Reply:
Without extensions `krita` can't open files. That's why I use `.ext` to underline that extension is mandatory.

Reply:
The `path/to/file` is a placeholder for any kind of filename, with extension or not. Krita doesn't need an explicit extension to recognize the file.

Reply:
Just tested. Yes, you are right, without extensions everything works. ;)